index.html

```diff
@@ -28,6 +28,142 @@ <h1>Iheagwara<br>Augustine</h1>
         <button class="Start-collab">Start collaboration</button>
         </section>            
         <div class="image-1-wrapper"><div class="image-1"></div></div>
+        <section class="section-2">
+            <div class="projects"><h2 >Projects</h2></div>
+            <div class="project-items-1">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
```

Comment:
- [ ] Please as the last reviewer said use a grid to align the whole work section and use flex to align the items and the card them selves, the reason for this is that the grid is better for two-directional aligning, and when you move to the desktop version will need a grid to align the while the flex is better for the one-directional aligning so the things you are trying to align the flex which should b simple.
![image](https://user-images.githubusercontent.com/74744787/212512566-493750a5-0d6c-4e3f-8fe4-a595d54157d6.png)

Reply:
hey there's flex on the card the wrapper is called languages @MeqdamAlqudah all i should do now is the grid right?
please check

Reply:
> hey there's flex on the card the wrapper is called languages @MeqdamAlqudah all i should do now is the grid right? please check

you should use flex in each card and use grid in the section tag instead of flex

Reply:
@MeqdamAlqudah  i just made a recent change please verify this meets the criteria